frontend/src/components/pages/CreateReview/AddNumberInput.tsx

```diff
@@ -13,7 +13,7 @@ type AddNumberInputProps = {
   mb?: number;
   minNum: number;
   maxNum: number;
-  numberInputFieldValue: number;
+  numberInputFieldValue: number | "";
```

Comment:
Is there a reason this is part of the PR? (Maybe you just need to rebase :thinking: )